Taskfile.yml

```
@@ -131,3 +131,10 @@ tasks:
       - task: install
       - task: generate
       - task: build
+
+  clear-accounts:
+    cmds:
+      - cmd: rm ~/Library/Application\ Support/massa-station-wallet/wallet*.yaml
```

Shouldn't we add the `_` on darwin too ? 
```suggestion
      - cmd: rm ~/Library/Application\ Support/massa-station-wallet/wallet_*.yaml
```